Simple-Projects/Simple_Projects/Basic/List/List.h

```diff
@@ -14,7 +13,7 @@ class List {
         double multiplier;
         int current;
     public:
-        List(int capacity=100, double multiplier=1.05);
+        List(int capacity=8, double multiplier=1.5);
```

Comment:
The multiplier of 1.05 makes the amortized constant insertion time almost nonexistent - when there are many reallocations the object will have to use realloc almost all the time.
The default capacity of 100 is too much for default initialization as default-constructed Lists will start out as empty and there are often arrays that do not have very many objects. That way, arrays with like ~10 elements will take up like 10 times much space. Just let it grow as needed with the multiplier, or provide the greater capacity when it is definitely needed.